cdap-app-fabric/src/main/java/co/cask/cdap/gateway/handlers/ProfileHttpHandler.java

```diff
@@ -120,7 +130,8 @@ public void writeProfile(HttpRequest request, HttpResponder responder,
   public void deleteProfile(HttpRequest request, HttpResponder responder,
                             @PathParam("namespace-id") String namespaceId,
                             @PathParam("profile-name") String profileName) throws NotFoundException, ConflictException {
-    // TODO: implement the method
+    // TODO: implement the method, though we have ProfileStore, we have to make sure the profile is not associated to
+    // any program and schedules before deleting
```

Comment:
i think its fine to just delete it for now and keep this todo here. It will help UI work.